scripts/aws-copy.sh

```diff
@@ -49,6 +49,20 @@ function wait_for_image() {
   log "Copy operation in region $REGION for image $AMI_IN_REGION finished"
 }
 
+function check_if_image_is_public_and_available_in_region() {
+  REGION=$1
+  AMI_IN_REGION=$2
+  IMAGE_DESC=$(aws ec2 describe-images --region $REGION --image-ids $AMI_IN_REGION)
+  REGEX_PUBLIC="Public: true"
+  REGEX_STATE="State: available"
+  if [[ $IMAGE_DESC =~ $REGEX_PUBLIC ]] && [[ $IMAGE_DESC =~ $REGEX_STATE ]]; then
+    echo "The $AMI_IN_REGION is PUBLIC and AVAILABLE in $REGION region."
+  else
+    echo "FAILURE | The $AMI_IN_REGION in $REGION region is not available or not in correct state."
```

Comment:
I think separate error messages would be useful for not available and not public images

Reply:
Also, please use `log` instead of `echo` to include the current timestamp